Source/WebCore/Modules/ShapeDetection/Implementation/Cocoa/VisionUtilities.mm

```diff
@@ -79,15 +80,15 @@ void configureRequestToUseCPUOrGPU(VNRequest *request)
     for (VNComputeStage computeStage in supportedComputeStageDevices) {
         bool set = false;
         for (id<MLComputeDeviceProtocol> device in supportedComputeStageDevices[computeStage]) {
-            if ([device isKindOfClass:MLGPUComputeDevice.class]) {
+            if ([device isKindOfClass:[PAL::getMLGPUComputeDeviceClass() class]]) {
```

Comment:
I think the `+class` here is redundant?

Reply:
it isn't, sadly :(

Reply:
Hmm, I see a bunch of `isKindOfClass:PAL::get` in the code. Should we be concerned...

Reply:
Interesting! Let me try again! I assumed it wasn't redundant, because the previous code said `MLGPUComputeDevice class` but maybe the new way just works!

Reply:
Yeah, it should just work.

The `get...Class()` is really just a wrapper (that also caches the return value) for `objc_getClass`, which returns a `Class`. (see `wtf/cocoa/SoftLinking.h`)

The only difference here would be if `MLGPUComputeDevice` were to override `+class`. But I think that would just be asking for trouble 😅

Reply:
It compiles! Let me see if it does what it's supposed to do...

Reply:
Yes! It works! \o/